Mathlib/LinearAlgebra/Matrix/NonsingularInverse.lean

```diff
@@ -483,6 +483,37 @@ theorem inv_eq_left_inv (h : B * A = 1) : A⁻¹ = B :=
 theorem inv_eq_right_inv (h : A * B = 1) : A⁻¹ = B :=
   inv_eq_left_inv (mul_eq_one_comm.2 h)
 
+section Woodbury
+
+variable [Fintype m] [DecidableEq m]
+variable (A: Matrix n n α) (U : Matrix n m α) (C : Matrix m m α) (V : Matrix m n α)
+variable [Invertible A] [Invertible C][Invertible (C⁻¹ + V * A⁻¹* U)]
+
+lemma add_mul_mul_inv_mul_eq_one: (A+U*C*V)*(A⁻¹-A⁻¹*U*(C⁻¹+V*A⁻¹*U)⁻¹*V*A⁻¹) = 1 := by
```

Comment:
```suggestion
lemma add_mul_mul_inv_mul_eq_one: (A + U * C * V) * (A⁻¹ - A⁻¹ * U * (C⁻¹ + V * A⁻¹ * U)⁻¹ * V * A⁻¹) = 1 := by
```
Can you also properly space out the rest of the proof?

Reply:
I think it might be worth deviating from the "spaces aroung `*`" style here, and only adding them around the additive operators.